### ISSUE TITLE
 UnoEuro changed its name to Simply.com

### ISSUE THREAD

**Issue by emilstahl** [acmesh-official/acme.sh#2906]
Hi

I would like to update [dns_unoeuro.sh](https://github.com/acmesh-official/acme.sh/blob/master/dnsapi/dns_unoeuro.sh) as UnoEuro recently changed its name to Simply.com. New API hostname is api.simply.com.

I don't want to break any existing use of the UnoEuro/Simply.com DNS plugin – what is recommended way? Create a new files named dns_simplycom.sh with updated content and keep the dns_unoeuro.sh file and just update the name and hostname in the existing file? Or can I rename the file without breaking current use of the DNS plugin?

**Comment by Neilpang** [comment]
>>Create a new files named dns_simplycom.sh with updated content and keep the dns_unoeuro.sh file and just update the name and hostname in the existing file? Or can I rename the file without breaking current use of the DNS plugin?


Yes, please create a new file, with the identical file content, except renaming `UNO_Key` and `UNO_User` to meet the new api name.


And please also udpate the url in the old  dns_unoeuro.sh, so that it will work when someone is renewing the cert.

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.

**Comment by neilpang** [comment]
Fixed in 65c06da2 (2020-11): dns_simply.sh was added for Simply.com; dns_unoeuro remains for existing users.
